### PR TITLE
[DOCS-492] Updates to troubleshooting tips

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Troubleshoot Sign-in Issues > Your Account Isn’t Fully Set Up | [Link](https://deploy-preview-392--cobalt-docs.netlify.app/platform-deep-dive/cobalt-account/account-recovery/#your-account-isnt-fully-set-up) | |
| Troubleshoot Sign-in Issues > Your Email or Password is Invalid | [Link](https://deploy-preview-392--cobalt-docs.netlify.app/platform-deep-dive/cobalt-account/account-recovery/#your-email-or-password-is-invalid) | |
| Troubleshoot Sign-in Issues > Error Messages upon Signing In | [Link](https://deploy-preview-392--cobalt-docs.netlify.app/platform-deep-dive/cobalt-account/account-recovery/#error-messages-upon-signing-in) | |
| Troubleshoot Your SAML SSO Configuration | [Link](https://deploy-preview-392--cobalt-docs.netlify.app/platform-deep-dive/organization/organization-settings/saml-sso/#troubleshoot-your-saml-sso-configuration) | |
| Notifications > Troubleshoot Email Notifications | [Link](https://deploy-preview-392--cobalt-docs.netlify.app/platform-deep-dive/collaboration/manage-notifications/#troubleshoot-email-notifications) | |
| Glossary > Sender Policy Framework | [Link](https://deploy-preview-392--cobalt-docs.netlify.app/getting-started/glossary/#sender-policy-framework) | |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Cobalt Account/account-recovery.md [hunk 1]
```diff
@@ -86,6 +88,33 @@ To reset your password:
...
+  - If you're an Organization Owner: To resend an invitation to a user, [remove them from your organization](/platform-deep-dive/organization/manage-users/#remove-users), and then [invite](/platform-deep-dive/organization/manage-users/#invite-users) them again.
+  - If you're the only Organization Owner in your organization: To get help, contact your Customer Success Manager (CSM) or support@cobalt.io.
+
+## Your Email or Password is Invalid
+
+When you sign in, you may see an error stating that your email or password is invalid.
+
+Ensure that:
+
+- You're using the correct email address that you registered with.
```
Comment (molfinn) [inline content/en/Platform Deep Dive/Cobalt Account/account-recovery.md:112]: Do we need to be super clear here? And say something like Ensure that you're email has been invited?

File: content/en/Platform Deep Dive/Cobalt Account/account-recovery.md [hunk 2]
```diff
@@ -161,6 +190,6 @@ If you have access to your Cobalt account, do the following:
...
+| `You have to confirm your email address before continuing` | You haven't fully set up your account on the Cobalt platform. | Confirm your email address by clicking the link in the email invitation, and then create a password. Learn more about [setting up your account](#your-account-isnt-fully-set-up). |
```
Comment (molfinn) [inline content/en/Platform Deep Dive/Cobalt Account/account-recovery.md:195]: Should we add the invalid email or password error here?

Comment (molfinn) [reply content/en/Platform Deep Dive/Cobalt Account/account-recovery.md:195]: Option are to make sure emails correct and is registered. Reset your password using the forgot password flow.

**Review by molfinn — APPROVED** [review]

File: content/en/Platform Deep Dive/Cobalt Account/account-recovery.md (hunk 2 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Cobalt Account/account-recovery.md]: Added this error. We'll redirect users to the corresponding section on this page.

File: content/en/Platform Deep Dive/Cobalt Account/account-recovery.md (hunk 1 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Cobalt Account/account-recovery.md]: Updated this.

**Pull request merged**